docs/tutorials/Build_Two_Parties_FATE_Cluster_in_One_Linux_Machine_with_MiniKube_zh.md

```diff
@@ -227,94 +250,156 @@ podSecurityPolicy:
   enabled: false
 ingressClassName: nginx
 modules:
-  - rollsite
-  - clustermanager
-  - nodemanager
-  - mysql
   - python
+  - mysql
   - fateboard
   - client
+  - spark
+  - hdfs
+  - nginx
+  - pulsar
 
-backend: eggroll
+computing: Spark
+federation: Pulsar
+storage: HDFS
+algorithm: Basic
+device: CPU
 
 ingress:
   fateboard:
     hosts:
     - name: party9999.fateboard.example.com
-  client:  
+  client:
     hosts:
     - name: party9999.notebook.example.com
-
-rollsite:
-  type: NodePort
-  nodePort: 30091
-  partyList:
-  - partyId: 10000
-    partyIp: 192.168.100.123
-    partyPort: 30101
+  spark:
+    hosts:
+    - name: party9999.spark.example.com
+  pulsar:
+    hosts:
+    - name: party9999.pulsar.example.com
 
 python:
   type: NodePort
   httpNodePort: 30097
   grpcNodePort: 30092
   logLevel: INFO
 
-servingIp: 192.168.100.123
+servingIp: 10.182.137.144
```

Comment:
Please 192.168.xxx.xxx, because there is a security check in Webank to verify if the fake IP in document.